tests/test_ansible.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import unittest
+import shlex
+from helper import shell_process
+import re
+
+"""Moneo ansible single node test"""
+
+
+class AnsibleTestCase(unittest.TestCase):
+    """Ansible unit test class"""
+    
+    def check_docker_status(self):
+        """Helper to check docker container status"""
+        cmd='sudo docker container ls'
+        args = shlex.split(cmd)
+        result = shell_process.shell_cmd(args)
+        if ('grafana' in result) or ( 'prometheus' in result ):
```

Comment:
Do you think it would be worth splitting the check for grafana and prometheus? It may make it easier to identify problems that may arise.

Reply:
Considering that the check is only to see if they are both containers up. I am not sure it warrants breaking it up into 2 assert statements or functions. Let me think about this one.